.github/workflows/qase-patch-validation-create.yaml

```diff
@@ -0,0 +1,33 @@
+name: Qase Patch Validation Create
```

Comment:
`Create Test Plan in Qase`, Suggest to remove patch validation if we are using the same for minor

Reply:
Create Patch Validation Test run as this is specific to patch validation. For minor we will need to create a separate test run